src/cli/dev.test.ts

```diff
@@ -161,7 +161,7 @@ describe("dev command", () => {
     const debugSpy = vi.spyOn(console, "debug").mockImplementation(() => {});
     const processOnSpy = vi.spyOn(process, "on");
 
-    await program.parseAsync(["node", "test", "dev", "--yes"]);
+    await program.parseAsync(["npx", "pepr", "dev", "--yes"]);
```

Comment:
```suggestion
    await program.parseAsync(["dev", "--yes"], { from: "user" });
```

Consistency with other tests we've written, though it's functionally equivalent